aiosmtpd/tests/test_server.py

```diff
@@ -155,19 +155,6 @@ def test_unknown_args(self):
         finally:
             cont.stop()
 
-    def test_conflict_smtputf8(self):
-        kwargs = dict(enable_SMTPUTF8=True)
-        cont = Controller(Sink(), server_kwargs=kwargs)
-        try:
-            with self.assertRaises(TypeError) as cm:
-                cont.start()
-            self.assertIsNone(cont.smtpd)
-            excm = str(cm.exception)
-            self.assertIn("multiple values", excm)
-            self.assertIn("enable_SMTPUTF8", excm)
-        finally:
-            cont.stop()
-
```

Comment:
Because `enable_SMTPUTF8` has been injected into `server_kwargs`, conflict will never happen, so this test is obsolete.